CLAUDE.md

```
@@ -0,0 +1,629 @@
+```asciidoc
+[source,terminal]
+----
+$ hcp create cluster <platform> --help 
+----
++
+Use the `hcp create cluster` command to create and manage hosted clusters. The supported platforms are `aws`, `agent`, and `kubevirt`.
+```
+
+**For multiple parameters/variables**
+* Use a definition list to explain multiple options, parameters, user-replaced values, placeholders, or UI elements.
+**  List the parameters or variables in the order in which they appear in the code block.
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)